docs/raw/serving-community-history.md

```diff
@@ -0,0 +1,315 @@
+- [Message archive distribution](#message-archive-distribution)
+- [Canonical message histories](#canonical-message-histories)
+- [Fetching messsage archives](#fetching-message-archives)
+  - [Downloading message archive indices](#downloading-message-archive-indices)
+  - [Downloading individual archives](#downloadinng-individual-archives)
+- [Storing historical messages](#storing-historical-messages)
+- [Considerations](#considerations)
+  - [Bandwidth consumption](#bandwidth-consumption)
+  - [Multiple community owners](#multiple-community-owners)
+
+## Terminology
+
+The following terminology is used throughout this specification:
+
+| Name                 | References |
+| -------------------- | --- |
+| Waku node            | An Ethereum node with Waku V1 enabled, or a [10/WAKU2](https://rfc.vac.dev/spec/10/) node that implements [11/WAKU2-RELAY](https://rfc.vac.dev/spec/11/)|
+| Store node           | A Waku node that implements [11/WAKU-MAILSERVER](/spec/11) or [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) respectively |
+| Waku network         | A group of Waku nodes connected through the internet connection and forming a graph |
+| Community owner      | A Status user that owns a Status community |
+| Community member     | A Status user that is part of a Status community |
+| Community owner node | A Status node with message archive capabilities enabled, run by a community owner |
+| Community member node| A Status node with message archive capabilities enabled, run by a community member |
+| Live messages        | Waku messages received through the Waku network |
+
+
+## Supported platforms
+
+Creating, distributing, downloading and unpacking community history archives SHOULD only be supported in desktop clients. Mobile clients SHOULD NOT implement this functionality due to bandwidth and storage limitations.
+
+## Requirements / Assumptions
+
+This specification has the following assumptions:
+
+- Store nodes are available 24/7, ensuring constant live message availability
+- Store nodes have enough storage to persist historical messages for up to 30 days
+- No store nodes have storage to persist historical messages older than 30 days
+- All nodes are honest
+- The network is reliable
+
+Furthermore, it assumes that:
+
+- Community owner nodes have enough storage to persist historical messages older than 30 days
+- Community owner nodes provide archives with historical messages **at least** every 30 days
+- Community owner nodes receive all community messages
+- Community owner nodes are honest
+
+## Overview
+
+The following is a high-level overview of the user flow and features this specification describes.
+
+### Serving community history archives
+
+Community owner nodes go through the following (high level) process to provide community members with message histories (assumes community owner node is available 24/7):
+
+1. Community owner creates a Status community
+2. Community owner enables community history archive support
```

Comment:
I think community history archive support should be on by default when a community owner creates a new community

Reply:
We will allow community members to turn this feature off, so I assume there's gonna be some UI switch that they can use when creating/editing communities.

I don't think we want to **force** community owners to have this enabled. Which means, owner could switch it off during creation. Because I assume that, I made it explicit that it has been enabled.

Reply:
Yes, Community Owners should definitely be able to switch the community history archive service off, we will have a toggle in the community admin settings to let a community owner do this.  I was proposing that this service should be switched on by default when creating a new community.